Client/metrics.yaml

```
@@ -2530,6 +2530,20 @@ tabs:
       - fx-ios-data-stewards@mozilla.com
     expires: "2023-12-01"
     unit: quantity of private tabs
+  inactive_tabs_count:
+    type: quantity
+    description: |
+      A quantity that indicates how many INACTIVE tabs a user
+      currently has open. This is reported when the user backgrounds
+      the app.
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/14906
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/14906
```

I think we should put the data review location on that link instead of the issue, since the data review happens there
```suggestion
      - https://github.com/mozilla-mobile/firefox-ios/pull/15511/
```

---

Thanks Laurie! Yup this is on the list to do before this PR is marked ready to review.